### DISCUSSION TITLE
Difference between app-play.apk and app-release.apk

### DISCUSSION THREAD

**Discussion by moiri-gamboni** [spacecowboy/Feeder#183] [Q&A] [answered]
Hi,
As the title says, what's the difference between the two versions on the releases page?
Thank you for all your work.

**Comment by spacecowboy** [comment, marked as answer]
the package name.

you can install them in parallel. fdroid uses the release version

**Reply by tempneq2000** [reply]
Hi. Sorry to bother. As far as I know, the two builds are completely identical. Same code, same functionality, same build flags, same everything except app ID. F-droid even provides both versions. So is there even a reason to maintain two different builds? It seems to cause more confusion and complexity than it should.

**Reply by spacecowboy** [reply]
There is a difference in package name because a scammer uploaded a copy of the app with ads before I published it in the play store. I got play to take the app down but they would not grant me ownership of the package name.

I have always considered F-Droid the primary distribution channel and the app is only on the Play store to stop scammers.

I am re-using the play store version to offer reproducible build in F-Droid faster than F-Droid themselves can build it

**Reply by tempneq2000** [reply]
Understood. I'm really sorry to hear such atrocity has happened to you. Btw, in that case, perhaps you could stop maintaining the "normal" build, and publish only the com.nononsenseapps.feeder.play appID build across all distribution platforms?

**Reply by spacecowboy** [reply]
No, then all the people using the real id won't get any updates.

Just pick one and be happy :)

**Reply by tempneq2000** [reply]
Yeah that's valid. However, I'd suggest you put a pop-up notice in the redundant build at app launch to tell people to switch to the other build with the Github & Fdroid download links given. I don't think people who specifically download their apps from non-Google Play sources would be reluctant or lazy enough to switch an app build. But I get that it's totally up to your decision
